tests/common/devices.py

```diff
@@ -758,9 +758,13 @@ def get_feature_status(self):
             bool: status obtained successfully (True | False)
         """
         feature_status = {}
-        command_output = self.shell('show features', module_ignore_errors=True)
-        if command_output['rc'] != 0:
-            return feature_status, False
+        command_list = ['show feature status', 'show features']
+        for cmd in command_list:
+            command_output = self.shell(cmd, module_ignore_errors=True)
+            if command_output['rc'] == 0:
+                break
+            else:
```

Comment:
This 'else' should pair with the 'for' construct not the 'if' construct

Reply:
Thanks Neetha. It's a bug. I'll make an update.